docs/hello_nextflow/02_hello_world.md

```diff
@@ -596,17 +709,25 @@ The output is simply the uppercase version of the text string:
 HELLO WORLD
 ```
 
-#### 2. Make the command take a file as input and write the output to a file
+!!! note
+
+    This is a very naive text replacement one-liner that does not account for accented letters, so for example 'Holà' will become 'HOLà'. This is expected.
+
+### 7.2. Make the command take a file as input and write the output to a file
+
+As previously, we want to output results to a dedicated file, which we name by prepending the original filename with `UPPER-`.
```

Comment:
How about changing the command below to:

```
cat results/output.txt | tr '[a-z]' '[A-Z]' > UPPER-output.txt
```
?

output.txt does not exist at the root, which might confuse complete CLI novices.